sessions/recap-project-5/example-implementation/pages/art-pieces/index.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import ArtPiecesPreview from "../../components/ArtPiecesPreview";
+
+const ImageGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(1fr);
+  grid-gap: 5px;
+  width: 600px;
+  height: 100%;
+`;
+
+export default function ArtPieces({
+  pieces,
+  handleClickToggleFavorite,
+  artPiecesInfo,
+}) {
+  return (
+    <>
+      <ImageGrid>
```

Comment:
We should move the renderring / mapping and the Styled component to separate component `ArtPieces`.


This way we can reuse the same component for the list of all pieces and the favorite pieces without having to repeat ourself.

The Next.js apge can do the filtering on the favorites page and pass all items to render via props